src/eureka/S1_detector_processing/s1_process.py

```diff
@@ -90,9 +90,9 @@ def rampfitJWST(eventlabel, ecf_path=None):
             if hdulist[0].header['INSTRUME'] == 'NIRCAM':
                 hdulist[0].header['NDITHPTS'] = 1
                 hdulist[0].header['NRIMDTPT'] = 1
-
+            
             EurekaS1Pipeline().run_eurekaS1(filename, meta, log)
-
+        
```

Comment:
If you want to add blank lines here, please make sure they have no tabs or spaces on them